packages/ui/core-components/src/lib/unsorted/viz/references/ReferenceArea/reference-area.d.ts

```diff
@@ -0,0 +1,61 @@
+// @ts-check
+
+import { Writable, Readable } from 'svelte/store';
+import type { MarkAreaComponentOption } from 'echarts';
+
+import type { Color } from '../colors.js';
```

Comment:
Not sure about just calling this `Color` - it isn't specific here, maybe `ReferenceColor`?

Reply:
[6e5ddf9](https://github.com/evidence-dev/evidence/pull/2184/commits/6e5ddf96090176b0f3aabae9bf971072ea90ceff)